@here/harp-mapview/lib/DataSource.ts

```diff
@@ -195,6 +196,18 @@ export abstract class DataSource extends THREE.EventDispatcher {
      */
     abstract getTilingScheme(): TilingScheme;
 
+    /**
+     * Returns a [[ElevationRangeSource]] so that `MapView` can compute proper bounding boxes that
```

Comment:
"MapView can compute" is incorrect, this should be VisibleTileSet?

Perhaps mentioning where it is computed is not needed.

Reply:
VisibleTileSet is an internal thing but MapView and DataSource are public API that's why I put it like this.

Reply:
Good point.